scenarios/anomaly-detection/build.gradle

```diff
@@ -97,23 +91,31 @@ shadowJar {
     }
 }
 
+task scriptAnomalyDetection(type: CreateStartScripts) {
+    outputDir = file('build/scripts')
+    mainClassName = 'io.pravega.anomalydetection.ApplicationMain'
+    applicationName = 'anomaly-detection'
+    defaultJvmOpts = ["-Dlog4j.configuration=file:conf/logback.xml"]
+    classpath = files(jar.archivePath) + sourceSets.main.runtimeClasspath
+}
+
 distributions {
     main {
         baseName = archivesBaseName
         contents {
+            into('bin') {
+                from project.scriptAnomalyDetection
+            }
             into('lib') {
                 from shadowJar
                 from(project.configurations.shadow)
+                from(jar)
+                from(project.configurations.runtime)
+            }
+            into('conf') {
+                from(project.file('src/main/resources/logback.xml'))
```

Comment:
Doesn't seem right for the logback file to located in `src/main/resources/logback.xml` since it would then be embedded.

Reply:
According to your request, I moved the configuration files back to their original location (dist/conf). Note that I moved these files to the `resources` folder to be consistent with other applications that do the same. If you think that no application should embed configuration files within jars (according to our discussion in #112), then we can open an issue to change this in the rest of sample applications.